### PR TITLE
Feature/20180 use breadcrumbs on new appointment details pages

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
The details pages should display the standard breadcrumb across the top. Reduce the line height if word wrap happens for long breadcrumbs.

**How to test:**
select `Upcoming` dropdown option
click `Details` link for Community Care, VA Appointment, Express Care, Video Connect, 
repeat for  `Request` and  `Past` dropdown options
## Testing done
local and unit test

## Screenshots
Upcoming appt for **Community Care** details page
image

Upcoming for **VA appt** details page
image

Upcoming **Express Care** details page
image

Upcoming **Video Connect** details page
image

## Acceptance criteria
- [ ] replace the `Manage appointment` link with the standard breadcrumb for all the details pages in the homepage refresh 
- [ ] add whichever is appropriate `Appointment detail` or `Request detail` as the last breadcrumb text
- [ ] the last link in the breadcrumbs should be to the current page url.
- [ ] ensure the line height is appropriate if there is word wrap due to long breadcrumbs

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by vbahinwillit — COMMENTED** [review]
LGTM!

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx [hunk 1]
```diff
@@ -69,9 +70,9 @@ function CommunityCareAppointmentDetailsPage({
 
   return (
     <div>
-      <div className="vads-u-display--block vads-u-padding-y--2p5 vaos-hide-for-print">
-        ‹ <Link to="/">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx:74]: It's a little bit odd, but the last link in our breadcrumbs should be to the current page url. So the `to` prop here should point to `/community-care/${id}`. And the others would have slightly different urls.

File: src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx [hunk 2]
```diff
@@ -46,9 +47,9 @@ function ExpressCareDetailsPage({
 
   return (
     <PageLayout>
-      <div className="vads-u-display--block vads-u-padding-y--2p5">
-        ‹ <Link to="/">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx:51]: Might make sense to call this one "Request detail"

File: src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx [hunk 3]
```diff
@@ -94,9 +94,9 @@ function RequestedAppointmentDetailsPage({
 
   return (
     <div>
-      <div className="vads-u-display--block vads-u-padding-y--2p5">
-        ‹ <Link to="/requested">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx:98]: Similar to above, maybe "Request detail"

File: src/applications/vaos/appointment-list/sass/styles.scss [hunk 4]
```diff
@@ -82,6 +82,13 @@
   }
 }
 
+.vaos-appts__breadcrumbs{
+  padding: 2em 0;
+}
+.vaos-appts__breadcrumbs li {
+padding: 0em !important;
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/sass/styles.scss:89]: Missing some indentation here

**Comment by jenniemc** [comment]
@jbalboni, I had 3 files with merge conflicts so I pulled latest from master and merged into my local branch to ensure the refresh updates you made are working on locally. It may just be my branch but the express care refresh redirects me back to the landing page and community care fresh produce the message `We’re sorry. We’ve run into a problem`. Am I better off abandoning this branch and create a new branch from master? That way I'm assured I didn't corrupt your changes.

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx]: The last breadcrumb link now points to the same current page url but oddly it's inactive.

File: src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx (hunk 2 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx]: Good catch. I took the file name for its face value. I should have looked more closely into this detail page to examine that its an express care request detail and not appointment detail page.

File: src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx (hunk 3 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx]: Good catch. This makes total sense.

File: src/applications/vaos/appointment-list/sass/styles.scss (hunk 4 above)
Comment (jenniemc) [reply src/applications/vaos/appointment-list/sass/styles.scss]: Strange there is no linting warning over indentation

Comment (jbalboni) [reply src/applications/vaos/appointment-list/sass/styles.scss]: Yeah, I don't think the css linting is set up all that well in vets-website.

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx (hunk 1 above)
Comment (jbalboni) [reply src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx]: That's normal for the breadcrumbs. I don't remember the reasoning for it, though.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**